tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -84,7 +84,26 @@ $resourceTestCommands = @(
     @{Name = "Az.Resources [Tags]";           Command = {Get-AzureRmTag -ErrorAction Stop}},
     @{Name = "Az.TrafficManager";             Command = {Get-AzureRmTrafficManagerProfile -ErrorAction Stop}},
     @{Name = "Az.Billing [UsageAggregates]";  Command = {Get-UsageAggregates -ReportedStartTime '1/1/2018' -ReportedEndTime '1/2/2018' -ErrorAction Stop}},
-    @{Name = "Az.Websites";                   Command = {Get-AzureRmWebApp -ErrorAction Stop}}
+    @{Name = "Az.Websites";                   Command = {Get-AzureRmWebApp -ErrorAction Stop}},
+    @{Name = "Az.Accounts";                   Command = {Get-AzDomain -ErrorAction Stop}},
+    @{Name = "Az.Advisor";                    Command = {Get-AzAdvisorConfiguration -ErrorAction Stop}},
+    @{Name = "Az.DataBoxEdge";                Command = {try {Get-AzDataBoxEdgeBandwidthSchedule -ResourceGroupName $resourceGroupName -DeviceName 'smokeTestDevice' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.Databricks";                 Command = {try {Get-AzDatabricksVNetPeering -ResourceGroupName $resourceGroupName -WorkSpaceName 'smokeTestWorkSpace' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.DataShare";                  Command = {try {Get-AzDataShare -ResourceGroupName $resourceGroupName -AccountName 'smokeTestAccountName' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*NotFound*") {throw $_}}}},
```

Comment:
@Youri970410 Could you check the exported cmdlets inside each added module and find the simplest one (requires the least parameter) for smoke test

Reply:
Got it,Thanks for your reminder.